### ISSUE TITLE
encoding error: / => %2F

### ISSUE THREAD

**Issue by fragmentbyte** [murfman-de/slim-wiki#2]
![](http://i.imgur.com/txqfWhL.png?1)

**Comment by fragmentbyte** [comment]
In `Main.php` on line 212:

```
'path' => $hasContent ? urlencode($currentPath) : null,
```

change on

```
'path' => $hasContent ? urldecode($currentPath) : null,
```

**Comment by fragmentbyte** [comment]
Fix:  https://github.com/til-schneider/slim-wiki/pull/3

**Comment by til-schneider** [comment]
Your fix #3 would break special characters in directory names. I fixed all encoding problems with directory names (hopefully). See: 127b3bb03a0864dfb0132d18fe6c1764c0810c31

**Comment by fragmentbyte** [comment]
Thank you. I do not php programmer and do not know all the details.
